Source/Core/DolphinQt/Host.cpp

```diff
@@ -32,15 +21,11 @@ void* Host_GetRenderHandle()
 	return (void*)(g_main_window->GetRenderWidget()->winId());
 }
 
-void Host_RequestRenderWindowSize(int w, int h)
-{
-	DRenderWidget* render_widget = g_main_window->GetRenderWidget();
-	qApp->postEvent(render_widget, new QResizeEvent(QSize(w, h), render_widget->size()));
-}
+void Host_RequestRenderWindowSize(int w, int h) {}
 
 bool Host_RendererHasFocus()
 {
-	return g_main_window->RenderWidgetHasFocus();
+	return g_main_window->GetRenderWidget()->isActiveWindow();
```

Comment:
This doesn't always return true if the RenderWidget is a child of the main window instead of its own window, IME. I could be misremembering that, though.